packages/server-wallet/src/__chain-test__/directly-funded-channel.test.ts

```diff
@@ -86,9 +104,26 @@ beforeAll(async () => {
     })
   );
 
-  a = await Engine.create(aEngineConfig);
-  b = await Engine.create(bEngineConfig);
+  aEngine = await Engine.create(aEngineConfig);
+  bEngine = await Engine.create(bEngineConfig);
+  const aChainService = new ChainService({
+    ...aEngineConfig.chainServiceConfiguration,
+    logger: aEngine.logger,
+  });
+  const bChainService = new ChainService({
+    ...bEngineConfig.chainServiceConfiguration,
+    logger: bEngine.logger,
+  });
+
+  const syncOptions: SyncOptions = {
+    pollInterval: 1_000,
+    timeOutThreshold: 60_000,
+    staleThreshold: 10_000,
+  };
+  a = await Wallet.create(aEngine, aChainService, TestMessageService.create, syncOptions);
+  b = await Wallet.create(bEngine, bChainService, TestMessageService.create, syncOptions);
```

Comment:
do you see this being extracted as "common test setup" code?

Reply:
Yeah, I think that's pretty likely. Right now this setup is very similar to our "with peers" [tests](https://github.com/statechannels/statechannels/blob/eventless-engine/packages/server-wallet/jest/with-peers-setup-teardown.ts#L89:L89).

I think we should organize our tests a bit since right now we have:
- "with peers" tests that use **two** databases and a **mock** chain
- "direct-funding" tests that uses **two** databases and a **real** chain
- "chain" tests that use **one** database and a **real** chain
-  regular tests that use **one** and a **mock** chain